CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+# How to Contribute to Jaeger
+
+We'd love your help!
+
+Jaeger is [Apache 2.0 licensed](LICENSE) and accepts contributions via GitHub
+pull requests. This document outlines some of the conventions on development
+workflow, commit message formatting, contact points and other resources to make
+it easier to get your contribution accepted.
+
+We gratefully welcome improvements to documentation as well as to code.
+
+## Making A Change
```

Comment:
normally we include instructions in this file on how to get started, i.e. which setup/dependencies are required, how to run tests, etc. However, we could add this later, after this PR is merged. -- /cc @isaachier